hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,277 @@
+page**]({{< relref "/downloads" >}}).
+
+<!--more-->
+
+## Development Snapshots
+
+A _development snapshot_ can be considered an open-beta for an upcoming feature release. This means that it contains the latest and greatest features
+that were introduced since the last feature release. Note though that in contrast to a _release candidate_ the list of features that will be included
+in the final 1.4.0 release is _not_ yet frozen and is thus subject to change.
+
+This does also mean that these features have not been tested by a broad audience yet and are therefore likely to still contain bugs. These could be
+minor ones, but in theory these could also be major. Therefore it is important that you always back up your data (e.g. configuration files) if you
+have a previous version of Mumble installed.
+
+Due to the reasoning stated above, we do not recommend using snapshots in an environment where stability of the software is key (e.g. in a business
+context).
+
+That being said though, we strongly encourage you to try snapshots out if you are a tech-savy person that can deal with potentially arising problems
+and report any problems you encounter to our [**issue tracker**](https://github.com/mumble-voip/mumble/issues). This kind of feedback is very
+important for us to be able to make the actual release as stable and reliable a possible.
+
+
+## Feature spotlight
+
+As there are many, many changes with this new version, here is a selection of things that we think are most prominent. If you're interested in all
+changes, have a look at the full [changelog](#Changelog).
+
+### Channel listeners
+
+This feature can to some extent be compared to the ability to link channels. Except that this does not operate on a per-channel basis but on a
+per-user basis.
+
+Concretely this new feature allows a user to "listen to" a channel. In that case all audio that is heard by people in this particular channel (be it
+by direct communication, shouts or via linked channels) is also heard by the listening user. Conceptually you could picture "listening to" a channel
+with placing a microphone in a room and then listening to what is recorded.
+
+This feature is one-way only meaning that in order to speak to people in the listened channel, you'll have to either join the channel or shout to it.
+
+It should be noted though that you will not be secretly listened to with this new feature. Listening to a channel creates a new entry in the channel's
+user-list (referred to as an "ear" or a "listener") and is thereby just as visible as a user actually being in that channel. There is also a dedicated
+ACL and several new server config options that can restrict this new feature.
+
+Older clients (< 1.4.0) will not indicate listeners in the UI though! They are informed if this feature is enabled on a server when the join it though
+(via chat).
+
+Note: This feature requires server-side support and is thus only available on servers at least running Mumble 1.4.0.
```

Comment:
```suggestion
Note: This feature requires server-side support and is thus only available on servers that are at least 1.4.0.
```

Reply:
I actually prefer the original version for this one :eyes:

Reply:
In the original version we refer to Mumble as protocol, which is fine.

However, it may be confusing, as the server is called Murmur.

Reply:
Well in that issue on GitHub we decided that we will be referring to the server as "Mumble server" :point_up: 
And actually I don't think that this really is confusing since it does mention the word "server" in the sentence anyways :thinking: